packages/plugin-amplitude/src/androidMain/kotlin/ly.iterative.itly.amplitude/AmplitudePlugin.kt

```diff
@@ -35,12 +53,15 @@ actual class AmplitudePlugin actual constructor(
         amplitude.initialize(config.androidContext, apiKey)
     }
 
-    override fun identify(userId: String?, properties: Properties?) {
+    override fun identify(userId: String?, properties: Properties?, pluginCallOptions: AmplitudeIdentifyOptions?) {
```

Comment:
@dantetam can you please rename `pluginCallOptions` to just `options` to match the other SDKs.